go/enclave/db/storage.go

```diff
@@ -21,16 +22,23 @@ import (
 )
 
 type storageImpl struct {
-	db      ethdb.Database
-	stateDB state.Database
-	nodeID  uint64
+	db          ethdb.Database
+	stateDB     state.Database
+	nodeID      uint64
+	chainConfig *params.ChainConfig
```

Comment:
can you add a todo, to consolidate this chainconfig with the one used in the evm_facade?
Or maybe it can be done easily